packages/veritone-react-common/src/components/InfiniteWrapper/styles.scss

```diff
@@ -0,0 +1,4 @@
+.infinte-wrapper {
+  height: 300px;
```

Comment:
Do not use a static height. Height should be inherited from its parent and still work with the scroll loader